src/components/fields/AddressField.js

```diff
@@ -84,16 +65,8 @@ const AddressField = ({ address, autoFocus, editable, name, isValid, onChangeTex
         ref={ref}
         testID={testID}
         value={formatValue(inputValue)}
+        placeholder={android || isTinyPhone ? lang.t('fields.address.short_placeholder') : lang.t('fields.address.long_placeholder')}
       />
-      {!inputValue && (
```

Comment:
I didn't see the need for custom placeholder here, it looks exactly the same now.